docs/openshift/deploy.md

```diff
@@ -0,0 +1,102 @@
+# Deploy an OpenShift Cluster
+
+## Install Pre-requisites
+
+All the commands in this guide require both the Azure CLI and `acs-engine`. Follow the [installation instructions to download acs-engine before continuing](../acsengine.md#install-acs-engine) or [compile from source](../acsengine.md#build-from-source).
+
+For installation instructions see [the Azure CLI GitHub repository](https://github.com/Azure/azure-cli#installation) for the latest release.
+
+## Overview
+
+`acs-engine` reads a cluster definition which describes the size, shape, and configuration of your cluster. This guide takes the default configuration of one master and two Linux agents. If you would like to change the configuration, edit `examples/openshift.json` before continuing.
```

Comment:
specify that the linux agents/agent pools are for openshift infrastructure and end-user workloads